package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@x-govuk/edit-prototype-in-browser",
+  "version": "0.1.0",
+  "description": "An in-browser editor for the govuk-prototype-kit",
+  "main": "index.js",
+  "scripts": {
+    "test": "govuk-prototype-kit validate-plugin && standard"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/x-govuk/edit-prototype-in-browser.git"
+  },
+  "keywords": [
+    "govuk",
+    "prototype",
+    "kit",
+    "editor"
+  ],
+  "author": "Natalie Carey <natalie@nataliecarey.uk>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/x-govuk/edit-prototype-in-browser/issues"
+  },
+  "homepage": "https://github.com/x-govuk/edit-prototype-in-browser#readme",
+  "dependencies": {
+    "fs-extra": "^11.1.1",
+    "govuk-prototype-kit": "^13.8.2-rc.0",
```

Comment:
Is it better to wait until this has been released instead of depending on a release candidate? 🤔

Reply:
It's just to get the validator in, I'm comfortable with it as a short term thing while we're in a `0.1.0` version.